Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp

```diff
@@ -176,6 +176,7 @@ static ErrorOr<Optional<String>> generate_enum_initializer_for(StringView proper
         { "focus_policy"sv, "GUI::FocusPolicy"sv },
         { "foreground_role"sv, "Gfx::ColorRole"sv },
         { "frame_style"sv, "Gfx::FrameStyle"sv },
+        { "style"sv, "Gfx::FrameStyle"sv },
```

Comment:
If kept, this should be combined with the previous commit. Also, again, note the potential change from `style` to `frame_style`.